### PR TITLE
Issue 8

### LINKED ISSUE #25: [block] Product Comparison

### PR DESCRIPTION
**Pull request by varunmitra**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #25 

Test URLs:
- Before: https://main--airlessco--hlxsites.hlx.page/drafts/vmitra/productpage
- After: https://issue-8--airlessco--hlxsites.hlx.page/drafts/vmitra/productpage

### CODE REVIEW

**Review by dkuntze — CHANGES_REQUESTED** [review]
can you remove the .DS_Store files?

File: blocks/product-comparison/product-comparison.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    td = createTag('td');
+    const productImage = createTag('img', { class: 'product-cimage' });
+    productImage.setAttribute('src', element[0][Images]);
+    td.append(productImage);
+    tr.append(td);
+  });
+  thead.append(tr);
+  tr = createTag('tr');
+  td = createTag('td', { class: 'specheading' });
+  td.innerHTML = '<strong>SPECIFICATIONS</strong>';
```
Comment (dkuntze) [inline blocks/product-comparison/product-comparison.js]: Will this work with multiple languages?

**Review by dkuntze — CHANGES_REQUESTED** [review]

File: blocks/product-comparison/product-comparison.js (hunk 1 above)
Comment (varunmitra) [reply blocks/product-comparison/product-comparison.js]: resolved, fetching translated specification value

**Review by dkuntze — APPROVED** [review]
looks good. thanks for the changes

**Pull request merged**